stdlib/sys/file.gr

```diff
@@ -585,45 +585,20 @@ export let fdSync = (fd: FileDescriptor) => {
   }
 }
 
-let orderedFdflags = [
-  Append,
-  Dsync,
-  Nonblock,
-  Rsync,
-  Sync
-]
 
-let orderedRights = [
-  FdDatasync,
-  FdRead,
-  FdSeek,
-  FdSetFlags,
-  FdSync,
-  FdTell,
-  FdWrite,
-  FdAdvise,
-  FdAllocate,
-  PathCreateDirectory,
-  PathCreateFile,
-  PathLinkSource,
-  PathLinkTarget,
-  PathOpen,
-  FdReaddir,
-  PathReadlink,
-  PathRenameSource,
-  PathRenameTarget,
-  PathFilestats,
-  PathSetSize,
-  PathSetTimes,
-  FdFilestats,
-  FdSetSize,
-  FdSetTimes,
-  PathSymlink,
-  PathRemoveDirectory,
-  PathUnlinkFile,
-  PollFdReadwrite,
-  SockShutdown
-]
+let wasmSafeCons = (a, b) => {
+  // [TODO] Once grain-lang/grain#802 is fixed:
+  // Memory.incRef(WasmI32.fromGrain([...]))
+  Memory.incRef(WasmI32.fromGrain(cons))
+  Memory.incRef(WasmI32.fromGrain(a))
+  Memory.incRef(WasmI32.fromGrain(b))
+  cons(a, b)
+  // [TODO] Once grain-lang/grain#802 is fixed:
+  // [a, ...b]
+}
+
+let orderedFdflags = wasmSafeCons(Append, wasmSafeCons(Dsync, wasmSafeCons(Nonblock, wasmSafeCons(Rsync, wasmSafeCons(Sync, [])))))
```

Comment:
We are going to conflict on these, since I moved stuff around in this file in #792